src/v2/manifest/manifest_schema1.rs

```diff
@@ -49,4 +49,35 @@ impl ManifestSchema1Signed {
             .map(|l| l.blob_sum.clone())
             .collect()
     }
+
+    /// List all image labels stored in the history array of this manifest.
+    ///
+    /// The returned list has no ordering guarantees.
```

Comment:
I'm not sure what you wanted to communicate here. Is this a docker image property?

Reply:
I think I read this in the *serde_json* crate as a property of the *Object* which is a something like `BTreeMap<..>`, and the ordering is not specified there.